### PR TITLE
Feature/remote item selection

### PR DESCRIPTION
**Pull request by millerts**

### CODE REVIEW

File: pom.xml
Comment (millerts) [inline pom.xml]: I will clean up this file and point it to the latest versions.

File: student/pom.xml
Comment (millerts) [inline student/pom.xml]: I will clean up this file and point to latest versions once things are released and before merging.

File: student/src/main/java/tds/student/services/remote/RemoteAdaptiveService.java
Comment (millerts) [inline student/src/main/java/tds/student/services/remote/RemoteAdaptiveService.java]: I'm going to switch this to always return the page group created by the legacy code for now.  That way QA can verify the data is correct.  The PageGroup returned from this call is then used to insert items into the legacy database.

I could add assertions in here, which i had in before, but I think it is a bit overkill.  

@jtreuting - This is where the legacy and remote calls starts to break down.

File: student/src/test/java/tds/student/sql/repository/remote/RemoteExamItemResponseRepositoryTest.java
Comment (millerts) [inline student/src/test/java/tds/student/sql/repository/remote/RemoteExamItemResponseRepositoryTest.java:42]: This test was just testing that the restTemplate was called.  Fixed it to actually verify the call has the right data.

File: student/src/main/java/tds/student/services/remote/RemoteAdaptiveService.java
Comment (wtritch) [reply student/src/main/java/tds/student/services/remote/RemoteAdaptiveService.java]: Maybe just a `return legacyCallsEnabled ? pageGroup : remotePageGroup;` so we don't need a code change when we disable legacy?

**Review by wtritch — APPROVED** [review]
LGTM once the Pom is clean

File: pom.xml
Comment (wtritch) [inline pom.xml]: This is going to be changed back to a test-only dependency?

File: student/pom.xml
Comment (wtritch) [inline student/pom.xml:341]: test-scoped dependency?

Comment (millerts) [reply student/pom.xml:341]: Fixed all the assertj test scopes.  Nice catch.  I was using them to assert the values were the same in the remote and legacy objects.

**Pull request merged**
